File: 1431.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int c=0;
        List<Boolean> result = new ArrayList<>();
        for(int candy : candies){
            c = Math.max(candy, c);
        }
        for(int candy : candies){
            result.add(candy+extraCandies>=c);
        }
        return result;

    }
}

File: 1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        li = []
        maxi = 0

        for i in candies:
            if i > maxi:
                maxi = i

        for i in range(len(candies)):
            if (candies[i] + extraCandies) >= maxi:
                li.append(True)
            else:
                li.append(False)
        return li